tests/unit/mixins/relationship-support-serializer-test.js

```diff
@@ -90,7 +93,8 @@ module('Unit | Mixin | relationship-support-serializer', function (hooks) {
       },
     });
 
-    let UserSerializer = JSONAPIBagaaravelSerializer.extend(
+    let UserSerializer = JSONAPISerializer.extend(
+      JSONAPISerializerMixin,
```

Comment:
I like the idea, but I don't like the name. It doesn't describe that it's doing Bagaaravel specific tweaks. Maybe `BagaaravelSerializerMixin` would be a better name?

Reply:
Yeah I had the same idea about the name, `JSONAPIBagaaravelSerializerMixin` would be even more correct I think, but then I thought it might be stupid to add `Bagaaravel` again to the name since that's already part of the package name. But I agree `JSONAPIBagaaravelSerializerMixin` would be better if you only see the mixin name inside a file. I'll update!

Reply:
It's a default export, so maybe we can just document it like that? (and keep the file names as-is?)

Reply:
No, now you mentioned it, I will not be able to resist the urge to change/improve it.

Reply:
DUN!